src/main/java/dev/revere/alley/base/server/command/ServiceCommand.java

```diff
@@ -31,7 +31,9 @@ public void onCommand(CommandArgs command) {
                 "",
                 "&6&lService Commands",
                 " &f● &6/service menu &7| &fOpens the service menu.",
-                " &f● &6/service allowqueue &8<&7true/false&8> &7| &fAllow/disallow queueing.",
+                " &f● &6/service allowqueue &8(&7true/false&8) &7| &fAllow/disallow queueing.",
+                " &f● &6/service disablecrafting &7| &fDisable crafting for an item.",
```

Comment:
have one command to toggle instead maybe? this is just duplicated code for no reason